### ISSUE TITLE
we should add support for dispatching multiple actions.

### ISSUE THREAD

**Issue by amit1911** [nak2k/node-redux-shortcuts#11]
If i understand correctly right now we can only dispatch one action per keystroke set. We should add support to allow multiple actions to be dispatched.

we just need to change the following

```
export const bindShortcut = (keys, actionCreator) => dispatch => bind(keys, e =>
    dispatch(actionCreator()));
```

to

```
export const bindShortcut = (keys, actionCreators) => dispatch => bind(keys, e => {
    actionCreators.forEach((actionCreator) => dispatch(actionCreator()));
});
```

**Comment by nak2k** [comment]
Thank you for your interest in redux-shortcuts.

I have published redux-shortcuts@0.0.2 that supports for dispatching multiple actions.

**Comment by amit1911** [comment]
great :+1:
